keps/sig-storage/20190129-csi-migration.md

```diff
@@ -123,6 +127,53 @@ The detailed design was originally implemented as a [design proposal](https://gi
 
 * All volume operation paths covered by Migration Shim in Beta for >= 1 quarter without significant issues
 
+## Test Plan
+
+### Per-driver migration testing
+
+We will require each plugin/driver provider to set up public CI to run all existing
+in-tree plugin driver tests for their migrated driver.
+
+If migration is on for that plugin, we will add infrastructure that inspects CSI
+Drivers metrics to make sure that the driver is servicing the operations. We
+must also observe through metrics that none of the in-tree code is being called
```

Comment:
comment: Will have to think through carefully how not to make this flaky or false positives.